src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -254,6 +263,28 @@ Friend Module CompilationUtils
         Return c
     End Function
 
+#If TEST_OPERATION Then
+    Private Sub WalkOperationTrees(compilation As VisualBasicCompilation)
+        Dim operationWalker = TestOperationWalker.GetInstance()
+
+        For Each tree In compilation.SyntaxTrees
+            Dim semanticModel = compilation.GetSemanticModel(tree)
+            Dim root = tree.GetRoot()
+
+            ' need to check other operation root as well (property, etc)
+            For Each methodNode As MethodBlockSyntax In root.DescendantNodesAndSelf().Where(Function(n)
```

Comment:
Similar to C#, use TypeOf(Of MethodBlockBaseSyntax)() as the filter instead. This will add coverage for constructors, operators, ...